### ISSUE TITLE
valid-aria-role does not allow lists of roles

### ISSUE THREAD

**Issue by lencioni** [evcohen/eslint-plugin-jsx-a11y#5]
This rule considers the following JSX bad:

```jsx
function Foo() {
  return <div role="tabpanel row" />;
}
```

However, according to [role mapping](https://www.w3.org/TR/2014/REC-wai-aria-implementation-20140320/#mapping_role):

> WAI-ARIA allows multiple roles to be specified as an ordered set of space-separated valid role tokens. The additional roles are fallback roles similar to the concept of specifying multiple fonts in case the first choice font type is not supported.

**Comment by evcohen** [comment]
Got it...again, thanks for reporting. this supposed to make your life easier not harder 😃
